### ISSUE TITLE
Provide a better editing experience for antora.yml

### ISSUE THREAD

**Issue by danhaywood** [asciidoctor/asciidoctor-intellij-plugin#769]
## Why the new feature should be added

Opening up antora.yml shows a garbled preview, and there is no intellisense of modules.

![image](https://user-images.githubusercontent.com/1031625/120470964-1041e300-c39c-11eb-96e9-a95f2f513e34.png)

## How the new feature should work

Probably just suppress the preview, but provide some intellisense for nav.adoc's of any modules available.

**Comment by ahus1** [comment]
Thanks for reporting this, it's actually I bug ... it is opening the the preview editor instead of the YAML editor. Expect an updated release later today.

**Comment by ahus1** [comment]
Release 0.32.52 is available as of now, and it restores the YAML editor for antora.yml files. I broke that in a 0.31.49. There now the standard YAML editor again with YAML syntax highlighting and quick-docs.

I now change this to an enhancement again, as adding auto-complete for modules makes sense to add.

**Comment by ahus1** [comment]
Thanks for suggesting this feature, it is a good addition to the existing Antora functionality.

The latest pre-release 0.32.54 implements a first step for this. 

TL;DR: "For the keys _nav_ and _start_page_ in `antora.yml`, the plugin provides auto-completion."

The pre-release of the plugin is available from [GitHub releases](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/installing-a-preview-release.html) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html). Please provide feedback if you can.

The small screen recording shows this in action. 

![antora-component-descriptor-autocomplete](https://user-images.githubusercontent.com/3957921/120650477-2031f380-c47e-11eb-9b18-cb77b170dc09.gif)

Known issues: 
* if you try to auto-complete the _start_page_ after a module name like "ROOT:" without additional text following it, this doesn't work yet -- this seems to he due to an invalid YAML syntax resulting from the auto-complete. This will work if you quote the _start_page_
* the next pre-release will no longer suggest a _start_page_ beginning with "component::"

What already works: 
* Ctrl+Space for auto-complete
* renaming a file will rename the reference in antora.yml as well
* triggering auto-complete after entering a "/" in a _nav_ path

What is still missing: 
* automated tests for the new functionality
* validation for the keys _nav_ and _start_page_ if the file names don't resolve

**Comment by ahus1** [comment]
Pre-Release 0.32.55 is available that fixes a classloading/dependency issues in PHPStorm for this feature. Thank you for those who reported it via the automatic error reporting. 

The pre-release of the plugin is available from [GitHub releases](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/installing-a-preview-release.html) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).

**Comment by ahus1** [comment]
I've added a test for this, but no further code changes. 
The functionality that has been added in this ticket has been included in the regular release  0.32.56 and later, and is available from the JetBrains marketplace.

Thank @danhaywood for suggesting this nice feature!
